entrypoint.sh

```diff
@@ -30,6 +30,7 @@ if [  ${POCKET_CORE_S3_CONFIG_URL:-false} == false  ]; then
  	echo 'POCKET_CORE_S3_CONFIG_URL env variable not found, Using default configurations'
 	echo ${POCKET_CORE_SERVICE_WHITELIST:-[]} >  ${POCKET_PATH_DATADIR:-datadir}/service_whitelist.json
 	echo ${POCKET_CORE_DEVELOPER_WHITELIST:-[]} > ${POCKET_PATH_DATADIR:-datadir}/developer_whitelist.json
+	echo ${POCKET_CORE_CHAINS:-[]} > ${POCKET_PATH_DATADIR:-datadir}/chains.json
```

Comment:
Does this need to be changed for integration tests as well?

Reply:
No. None of the options above affects the integration tests